src/protocol/requests/listGroups/v3/response.js

```diff
@@ -0,0 +1,12 @@
+const responseV2 = require('../v2/response')
+
+/**
+ * ListGroups Response (Version: 3) => throttle_time_ms error_code [groups] TAG_BUFFER
+ *   throttle_time_ms => INT32
+ *   error_code => INT16
+ *   groups => group_id protocol_type TAG_BUFFER
```

Comment:
I haven't read about tagged fields yet, so I'm not sure if we need to do something with the `TAG_BUFFER` yet. Would be interesting to hear what it is and what it's for.

Reply:
Don't know, can it stay like this?

Reply:
Yeah, it's fine. It's likely an unrelated feature that we can. Implement separately later.